### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by BGillham** [niklasvh/html2canvas#2544] [Q&A]
The example on your website works great but I am a newbie and would like to have a downloadable example or a step-by-step way of implementing this great feature. Can anybody assist with working examples please?

**Comment by AgentSmith0** [comment]
You can find some examples here: https://github.com/niklasvh/html2canvas/tree/master/examples And the implementation is explained here:  https://html2canvas.hertzen.com/getting-started
